frontend/src/css/Sidebar.styl

```
@@ -93,7 +93,7 @@ spacingInsideMenu = 40px
       &:hover
         background-color var(--sidebarColorDarken)
     &__withoutChildren
-      padding-inline-start 20px
+      padding-inline-start 20px + standardSpacing
```

```suggestion
      padding-inline-start standardSpacing * 3
```
I think this brings more clarity in the code, and leaves no possibility to have non-standard spacing.